### ISSUE TITLE
Starting a navigation

### ISSUE THREAD

**Issue by i315315** [amosproj/amos2022ws02-automotive-test-app#4]
## User story
1. As a User
2. I want to simulate the start of a navigation 
3. So that {}

There should be a button that simulates the start of a navigation immediately. There should also be a possibility to simulate a delayed start (delay 30 seconds or one minute).

1. As a User
2. I want to be able to see, whether the app is currently simulating a navigation 
3. So that {}

## Acceptance criteria
* In the navigation context there is a possibility to simulate the start of a navigation immediately.
* In the navigation context there is a possibility to simulate the start of navigation after 30 seconds or after one minute.
* The delayed start of the navigation must also be done when the app is running in the background.
* The start of a navigation is correctly requested from the corresponding AOSP service.
* It should be visible in the app whether it currently has the focus of the navigation type.

## Notes
The necessary AOSP service is the CarAppFocusManager: https://developer.android.com/reference/android/car/CarAppFocusManager
The start of a navigation is simulated by requesting the focus at the CarAppFocusManager.
A listener can be registered at the CarAppFocusManager, which receives information about the current status of the app focus (addFocusListener()).

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by i315315** [comment]
Duplicate
